src/app/shared/components/advertising-area/advertising-area.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-advertising-area',
+  templateUrl: './advertising-area.component.html',
+  styleUrls: ['./advertising-area.component.scss']
+})
+export class AdvertisingAreaComponent implements OnInit {
+
+  contentHeight: number;
```

Comment:
height, width, and content should be Input properties

content is required, but height and width can be optional